conference_abstracts/RSECon24.md

```diff
@@ -0,0 +1,12 @@
+# Bring your own classroom to a CodeRefinery workshop
+
+CodeRefinery is a project teaching FAIR research software and development 
+and basic Research Software Engineering practices to students and researchers across all research fields. CodeRefinery is a collaboration between universities and organizations in the Nordics funded by Nordic e-infrastructure collaboration (NeiC).
+
+
+Our workshops are streamed and recorded so that anyone can follow them. Registered participants also have the possibility to interact with the instructors on stream via a collaborative document. The open source materials (available at https://coderefinery.org/lessons/core/) include lectures as well as hands-on activities. Sharing reusable training materials is one of the projects core values.
```

Comment:
```suggestion
Our workshops are streamed and recorded so that anyone can follow them. Registered participants also have the possibility to interact with the instructors on stream via a collaborative document. The open source materials (available at https://coderefinery.org/lessons/) include lectures as well as hands-on activities. Sharing reusable training materials is one of the projects core values.
```

Reply:
I would change that link to make all lessons more discoverable.